src/wp-includes/blocks/latest-posts.php

```diff
@@ -151,13 +151,13 @@ function render_block_core_latest_posts( $attributes ) {
 				$excerpt_length = (int) apply_filters( 'excerpt_length', $block_core_latest_posts_excerpt_length );
 				if ( $excerpt_length <= $block_core_latest_posts_excerpt_length ) {
 					$trimmed_excerpt  = substr( $trimmed_excerpt, 0, -11 );
+					$trimmed_excerpt .= '… <a href="' . esc_url( $post_link ) . '" rel="noopener noreferrer">';
 					$trimmed_excerpt .= sprintf(
-						/* translators: 1: A URL to a post, 2: The static string "Read more", 3: The post title only visible to screen readers. */
-						__( '… <a href="%1$s" rel="noopener noreferrer">%2$s<span class="screen-reader-text">: %3$s</span></a>' ),
-						esc_url( $post_link ),
-						__( 'Read more' ),
+						/* translators: %s: The post title only visible to screen readers */
+						__( 'Continue reading <span class="screen-reader-text">%s</span>' ),
 						esc_html( $title )
 					);
+					$trimmed_excerpt .= '</a>';
```

Comment:
@swissspidy, pinging you here as these changes have moved from https://github.com/WordPress/wordpress-develop/pull/5439#discussion_r1350597442 to here 🙇

Reply:
Again, no string concatenation please. The ellipsis AND the link tag should pe part of the translatable string.

Reply:
Strings targeted for screen readers have this comment in core:
`/* translators: Hidden accessibility text. */`
And combined with a placeholder, something like this:
`/* translators: Hidden accessibility text. %s: Post title */`

Reply:
Thanks! I've updated this in https://github.com/WordPress/wordpress-develop/pull/5441/commits/6e74cd9f95c0fa98475eaf4803dcf61256984d38 by removing the string concatenation and moving "Read more" into the translatable string. I think this is closer to your suggestions.

Reply:
Thanks for the feedback here, folks! And to @mikachan for updating the block 🙇🏻 

Just to provide context, the [original idea was to use the already-translated "Read more"](https://github.com/WordPress/wordpress-develop/pull/5259#discussion_r1346665397) - so thanks for the clarification on that approach.

I've got a [PR going in Gutenberg to sync the changes from this patch](https://github.com/WordPress/gutenberg/pull/55181). I'll give it a final update after this PR closes (should there be any further changes).